src/main/java/core/basesyntax/model/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.model;
+
+import java.util.Deque;
+import java.util.LinkedList;
+
+public class Storage {
+    public static Deque<FruitBox> storage = new LinkedList<>();
```

Comment:
let's use a different collection so that we could store different fruit, not only bananas
consider using `Map<String, Map<LocalDate, Integer>>`

this collection can be stored as a static field in your `StorageService`

Reply:
You can store different fruints in this store

Reply:
Тут можно хранить любые фрукты, просто, если у меня бананы с разным сроком годности, то они считаются как два разных объекта и попадают в storage (очередь) с учетом даты, те что испортятся раньше будут вначале. Я специально использовал очередь для этого, как мне кажется это логично для подобного задания

Reply:
okey, I get that you want them to be in order by expiration date , but what is the order between the different type of fruit in your case?

Reply:
@dmytro-hryhoruk Там у меня порядок только по дате. Есть коробка, на ней дата, она ставиться в очередь соответственно. При выводе всех элементов я проверяю совпадения по ключу и выводу все фрукты того или иного типа вне зависимости от даты (edited)